docs/post_installation/peering.md

```diff
@@ -24,21 +18,17 @@ Each time Hornet starts, the `PeerId` is written to stdout:
 2021-04-19T14:27:55Z  INFO    P2P     peer configured, ID: 12D3KooWEWunsQWGvSWYN2VR7wNNoHgae4XikBqwSre8K8sVTefu
 ```
 
-Your `PeerId` is an essential part of your `multiaddr` used to configure neighbors, such
-as `/dns/example.com/tcp/15600/p2p/12D3KooWHiPg9gzmy1cbTFAUekyLHQKQKvsKmhzB7NJ5xnhK4WKq`,
-where `12D3KooWHiPg9gzmy1cbTFAUekyLHQKQKvsKmhzB7NJ5xnhK4WKq`
-corresponds to your `PeerId`. Your `PeerId` is also visible on the start page of the dashboard.
+Your `PeerId` is an essential part of your `multiaddr` used to configure neighbors. For example,  `/dns/example.com/tcp/15600/p2p/12D3KooWHiPg9gzmy1cbTFAUekyLHQKQKvsKmhzB7NJ5xnhK4WKq`, where `12D3KooWHiPg9gzmy1cbTFAUekyLHQKQKvsKmhzB7NJ5xnhK4WKq` corresponds to your `PeerId`. Your `PeerId` is also visible on the start page of the dashboard.
 
-It is recommended however to pre-generate the identity, so you can pre-communicate it to your peers before you even
-start your node and also to retain the identity in case you delete your `./p2pstore` by accident.
+However, we recommended that you pre-generate the identity.  This way you can pre-communicate it to your peers before you even start your node, and you will be able to retain the identity in case you accidentally delete your `./p2pstore`.
```

Comment:
```suggestion
However, we recommended that you pre-generate the identity. This way you can pre-communicate it to your peers before you even start your node, and you will be able to retain the identity in case you accidentally delete your `./p2pstore`.
```